modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc

```diff
@@ -1,6 +1,13 @@
 [id="creating-a-telemetry-plugin_{context}"]
 = Creating A Telemetry Plug-in
 
+[WARNING]
+====
+This page explains how to create a back-end telemetry plug-in for {prod-short} workspaces, which will be deprecated in favour of {devworkspace}s.
```

Comment:
We can't write about the future in the docs.

Will removing this first sentence of this warning admonition make a difference to the meaning of the second sentence?

```suggestion
```

Reply:
In that case, could this sentence instead be shortened to:

"This page explains how to create a back-end telemetry plug-in for {prod-short} workspaces."

instead?

How does that sound @max-cx @ibuziuk ?

Reply:
This is how it would look like:
<img width="765" alt="image" src="https://user-images.githubusercontent.com/83611742/159714982-fae2a95a-17f5-4791-9bc9-870d0e758580.png">

Reply:
Let's leave it as is, the plan is to remove non-devworkspace part in the future PRs completely as part of https://github.com/eclipse/che/issues/21264 cleanup